### ISSUE TITLE
Mac not operating correctly when SIP is enabled after download

### ISSUE THREAD

**Issue by svee15** [pookjw/SidecarPatcher#56]
Since running the patch it works correctly when SIP is disabled. When I re-enable it, my Mac starts up to multiple apps crashing and icons missing from the menu bar (battery, WI-FI, etc.). 

Basically, when SIP is enabled after running the patch, my Mac is useless until I disable it again.

Any tips/advice please?

**Comment by IBOYHK** [comment]
I have the same issue

**Comment by teddybearrohan** [comment]
It is an issue on 10.15.4. For it to work on 10.15.4 you need to disable SIP and add amfi_get_out_of_my_way to your nvram bootflags

**Comment by DPr00f** [comment]
Yup, just tested on a macbook pro mid 2014, It stops working after reenabling csrutil and amfi_get_out_of_my_way

**Comment by IBOYHK** [comment]
yeah, I just felt like apple is limit too much, can't enable csrtil and side car 😔

**Comment by felsmann** [comment]
What are the risks of leaving SIP disabled

**Comment by mateo2905** [comment]
Hi guys! I am having the same issue! 
I just did the install and now it is broken. I will try everything I can or I will do a backup but I want to be aware if there are any fixes!

**Comment by cbarbiani** [comment]
Hey there, I have the same issue. After patch and re enable the SIP even finder is crashing. Anyone found a solution?

**Comment by felsmann** [comment]
Would this get overwritten when upgrading to the next OS?
